docs/docs/configuration/object_detectors.md

```diff
@@ -325,4 +334,60 @@ model:                                # required
   input_pixel_format: bgr             # required
   # shape of detection frame
   input_tensor: nhwc
-```
\ No newline at end of file
+```
+
+Explanation for rknn specific options:
+- **core mask** controls which cores of your NPU should be used. This option applies only to SoCs with a multicore NPU (at the time of writing this in only the RK3588/S). The easiest way is to pass the value as a binary number. To do so, use the prefix `0b` and write a `0` to disable a core and a `1` to enable a core, whereas the last digit coresponds to core0, the second last to core1, etc. You also have to use the cores in ascending order (so you can't use core0 and core2; but you can use core0 and core1). Enabling more cores can reduce the inference speed, especially when using bigger models (see section below). Examples:
+  - `core_mask: 0b000` or just `core_mask: 0` let the NPU decide which cores should be used. Default and recommended value.
+  - `core_mask: 0b001` use only core0.
+  - `core_mask: 0b011` use core0 and core1.
+  - `core_mask: 0b110` use core1 and core2. **This does not** work, since core0 is disabled.
+- **yolov8_rknn_model** see section below.
+- **min_score** sets the minimal detection confidence. Should have the same value as min_score in the object block. See also [Reducing false positives](/guides/false_positives.md).
+- **nms_thresh** is the IoU threshold for Non-maximum Suppression (NMS). Enable "bounding boxes" in the debug viewer.
+  - *Decrease* if two overlapping objects (for example one person in front of another) are detected as one object.
+  - *Increase* if there are multiple boxes around one object.
+
+### Choosing a model
+
+There are 5 yolov8 models that differ in size and therefore load the NPU more or less. In ascending order, with the top one being the smallest and least computationally intensive model:
+
+| Model   | Size in mb |
+| ------- | ---------- |
+| yolov8n | 9          |
+| yolov8s | 25         |
+| yolov8m | 54         |
+| yolov8l | 90         |
+| yolov8x | 136        |
+
+:::tip
+
+You can get the load of your NPU with the following command:
+
+```bash
+$ cat /sys/kernel/debug/rknpu/load
+>> NPU load:  Core0:  0%, Core1:  0%, Core2:  0%,
+```
+
+:::
+
+- By default the rknn detector uses the yolov8**n** model (`yolov8_rknn_model: n`). This model comes with the image, so no further steps than those mentioned above are necessary.
+- If you want to use are more precise model, you can set `yolov8_rknn_model:` to `s`, `m`, `l` or `x`. But additional steps are required:
+    1. Mount the directory `/model/download/` to your system using one of the below methods. Of course, you can change to destination folder.
```

Comment:
we should recommend users to use `/config/model_cache/rknn/` similar to how other detectors already do this